code/modules/nano/modules/human_appearance.dm

```diff
@@ -105,6 +106,11 @@
 			if(owner.set_accent(href_list["accent"]))
 				cut_and_generate_data()
 			return 1
+	if(href_list["language"])
+		if(can_change(APPEARANCE_GENDER) && (href_list["language"] in valid_languages))
```

Comment:
Tying the language (and previously the accent) to the APPEARANCE_GENDER flag is a ugly hack.
The language and accent should both be tried to a APPEARANCE_LANGUAGE flag that controls if the language can be edited or not.

Reply:
donezo